deps/chakrashim/include/v8-debug.h

```diff
@@ -91,7 +91,7 @@ class V8_EXPORT Debug {
   static Local<Context> GetDebugContext(Isolate* isolate);
 
   static void EnableDebug();
-  static void EnableInspector();
+  static void EnableInspector(bool enableReplayDebug = false);
```

Comment:
As far as I could see there was no `EnableDebug` function defined either, so I didn't worry about it.  The calls to both functions were gated behind CHAKRACORE conditions.

https://github.com/nodejs/node-chakracore/search?utf8=%E2%9C%93&q=enabledebug&type=